casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -715,6 +716,8 @@ object BlockAPI {
           "No action taken since other thread is already processing the block."
             .asLeft[String]
             .pure[F]
+        case BlockError.CasperIsBusy =>
+          s"Casper put block in the queue: $status".asLeft[String].pure[F]
```

Comment:
Can we merge `Processing` and `CasperIsBusy`? Seems like they both represent the same state.

Reply:
`Processing` actually means that block was already processed - it is returned when block acquired block and node found that this block was already aded. We should rename it I believe https://github.com/rchain/rchain/blob/10889ddd8976473853804cceeccaac4e292a8215/casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala#L202-L208

Reply:
Actually looks like `Processing` usage is not consistent 
https://github.com/rchain/rchain/blob/9e1cf7d9fcaeb098f22cc890a12abf9f4609eabd/casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala#L715-L718
here it means that block is in process of adding. But its odd to see it being used in createBlock